src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Action/ActionNotification.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo, type FC } from 'react';
+
+import {
+  type ColonyFragment,
+  useGetColonyActionQuery,
+  useGetUserByAddressQuery,
+} from '~gql';
+import { type Notification as NotificationInterface } from '~types/notifications.ts';
+
+import NotificationWrapper from '../NotificationWrapper.tsx';
+
+import ActionNotificationMessage from './ActionNotificationMessage.tsx';
+
+const displayName = 'common.Extensions.UserHub.partials.ActionNotification';
+
+interface NotificationProps {
+  colony: ColonyFragment | null | undefined;
+  loadingColony: boolean;
+  notification: NotificationInterface;
+}
+
+const ActionNotification: FC<NotificationProps> = ({
+  colony,
+  loadingColony,
+  notification,
+}) => {
+  const { data: userData, loading: loadingUser } = useGetUserByAddressQuery({
+    variables: { address: notification.customAttributes?.creator || '' },
+    skip: !notification.customAttributes?.creator,
+  });
+  const { data: actionData, loading: loadingAction } = useGetColonyActionQuery({
+    variables: {
+      transactionHash: notification.customAttributes?.transactionHash || '',
+    },
+    skip: !notification.customAttributes?.transactionHash,
+  });
```

Comment:
Would it make sense to extract the `creator` and `transactionHash` variables?
`const { creator: userAddress, transactionHash } = notification.customAttributes ?? {}` 
and then use them here

```
  const { data: userData, loading: loadingUser } = useGetUserByAddressQuery({
    variables: { address: userAddress || '' },
    skip: !userAddress,
  });
  const { data: actionData, loading: loadingAction } = useGetColonyActionQuery({
    variables: {
      transactionHash: transactionHash || '',
    },
    skip: !transactionHash,
  });
```